tst/testinstall/grpfp.tst

```diff
@@ -101,6 +101,17 @@ gap> Print( Collected( List( l, x -> Index( c2, x ) ) ), "\n" );
 gap> Print( Collected( List( LowIndexSubgroupsFpGroup( g, s, 5 ),
 >                            x -> Index( c2, x ) ) ), "\n" );
 [ [ 12, 1 ], [ 24, 1 ], [ 48, 1 ] ]
+
+# Tietze simplifications
+gap> F:=FreeGroup("a");;
+gap> SimplifiedFpGroup(F/[GeneratorsOfGroup(F)[1]]);
+<fp group on the generators [  ]>
+gap> F:=FreeGroup("a","b","c");;
+gap> rels:=ParseRelators(F,"a2,b3,c4,abC");
+[ a^2, b^3, c^4, a*b*c^-1 ]
+gap> IsomorphismSimplifiedFpGroup(F/rels);
+[ a, b, c ] -> [ c*b^-1, b, c ]
+
 gap> STOP_TEST( "grpfp.tst", 1);
```

Comment:
To let the tests pass, you need to either remove this empty line, or else insert a comment line right before the STOP_TEST line.

Reply:
As https://www.gap-system.org/Manuals/doc/ref/chap7.html#X87712F9D8732193C says:

> To allow for comments in fname the following lines are ignored by default: lines at the beginning of fname that start with "#", and one empty line together with one or more lines starting with "#".

Otherwise, it's assumed that the emtpy line is a part of the GAP output, hence the diff is reported.